orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/servergroup/TitusInterestingHealthProviderNamesProcessor.java

```diff
@@ -0,0 +1,44 @@
+package com.netflix.spinnaker.orca.clouddriver.tasks.servergroup;
+
+import com.netflix.spinnaker.orca.front50.model.Application;
+import com.netflix.spinnaker.orca.pipeline.model.Stage;
+import org.springframework.stereotype.Component;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@Component
+public class TitusInterestingHealthProviderNamesProcessor implements InterestingHealthProviderNamesProcessor {
+  private static final String TITUS = "Titus";
+  private static final String INTERESTING_HEALTH_PROVIDER_NAMES = "interestingHealthProviderNames";
+
+  @Override
+  public boolean supports(String cloudProvider, Application application, Stage stage) {
+    if (cloudProvider.equalsIgnoreCase(TITUS)) {
+      if (application.platformHealthOnly == Boolean.TRUE && application.platformHealthOnlyShowOverride == Boolean.TRUE) {
```

Comment:
I don't think you actually need this ... this `Processor` kicks in only if the current server group being acted upon has a label right?

Two things:
- Should it be restricted to a DeployStage w/ RollingPush
- Should it be looking at labels on the current server group (ie. fetching it from clouddriver) vs. what's in the `stage.getContext()`.